### DISCUSSION TITLE
启用svpflow-starlight之后的cpu负载问题

### DISCUSSION THREAD

**Discussion by topgaer** [hooke007/mpv_PlayKit#176] [Q&A 问答] [answered]
![Clip_20220910_141558](https://user-images.githubusercontent.com/52902386/189472238-53f5e2a5-d96e-425e-82a3-2e119b60f642.jpg)
默认情况就是这样，就是固定几个线程占用高，也不全是物理核心

我看到别人写的参考了一下把第二行注释掉改了

> import vapoursynth as vs
> #from vapoursynth import core ##默认的设置
> core = vs.core
> core.num_threads = 10 ##我自己修改的设置

结果cpu占用还是没变化

我的配置是5600X 6C12T 无核显 显卡是GTX 1070
svpflow-staarlight.vpy里面的GPU相关是默认的，启用/禁用补帧，显卡负载没有变化，是不是因为这里设置不对导致补帧没有调用到GPU加速？

> sp, ap, fp = """{
>         gpu: 1
>     }""", """{
>         block: { w: 32, h: 16, overlap: 2 },
>         main: {
>             levels: 5,
>             search: {
>                 type: 4, distance: -12,
>                 coarse: { type: 4, distance: -1, trymany: true, bad: { range: 0 } }
>             },
>             penalty: { lambda: 3.33, plevel: 1.33, lsad: 3300, pzero: 110, pnbour: 50 }
>         },
>         refine: [{ thsad: 400 }, { thsad: 200, search: { type: 4, distance: -4 } }]
>     }""", """{
>         gpuid: 0,
>         algo: 23, rate: { num: %d, den: %d, abs: true },
>         mask: { cover: 80, area: 30, area_sharp: 0.75 },
>         scene: { mode: 0, limits: { scene: 6000, zero: 100, blocks: 40 } }
>     }""" % (round(min(max(target_fps, fps * 2, freq / 2), freq)) * 1000, 1001)

但是我看了链接介绍
gpu: 1 启用gpu 
gpuid: 0使用第一张显卡，应该是对的吧

**Comment by hooke007** [comment, marked as answer]
from vapoursynth import core 和 core=vs.core 是等价物，等于没改

自动就能最大化利用线程数，少卡两线程也是等于没改

svp就是这样的，你要几乎百分百用gpu而不是cpu计算，用rife
